.zenodo.json

```diff
@@ -244,6 +244,9 @@
       "name": "P\u00e9rez-Garc\u00eda, Fernando",
       "orcid": "0000-0001-9090-3024"
     },
+    {
+      "name": "Braun, Henry"
```

Comment:
@hbraunDSP  If you would like to update your preferred publication name, your affiliation or ORCID, feel free to comment with those, or open a pull request against the `rel/2.5.1` branch.

Reply:
Thanks, I just opened a PR

Reply:
Thanks!